scalableminds/webknossos
# Fix undefined access on not-existing segmentation layer

### URL of deployed dev instance (used for testing):
- https://fixundefinedmesh.webknossos.xyz

### Steps to test:
- open a dataset without segmentation layer in view mode and switch to the `Meshes` tab (within a hybrid annotation, an segmentation layer would always exist)
- ensure that the console doesn't print `Uncaught (in promise) TypeError: UnhandledRejection: "TypeError: Cannot read property 'fallbackLayer' of null"`

### Issues:
- fixes airbrake reports I saw with `Uncaught (in promise) TypeError: UnhandledRejection: "TypeError: Cannot read property 'fallbackLayer' of null"`

------
